File: project/changzune-onboarding/src/main/java/com/innercircle/onboarding/changzune_onboarding/survey/repository/SurveyRepository.java
package com.innercircle.onboarding.changzune_onboarding.survey.repository;

import com.innercircle.onboarding.changzune_onboarding.survey.domain.Survey;
import org.springframework.data.jpa.repository.JpaRepository;

public interface SurveyRepository extends JpaRepository<Survey, Long> {
    // 필요 시 사용자 정의 쿼리 추가 가능
}

File: project/soojinhan-onboarding/src/main/kotlin/onboardingproject/project/repository/SurveyRepository.kt
package onboardingproject.project.repository

import onboardingproject.project.domain.Survey
import org.springframework.data.jpa.repository.JpaRepository

/**
 * packageName : onboardingproject.project.repository
 * fileName    : SurveyRepository
 * author      : hsj
 * date        : 2025. 6. 14.
 * description :
 */
interface SurveyRepository : JpaRepository<Survey, String>